zerver/webhooks/dialogflow/doc.md

```diff
@@ -0,0 +1,23 @@
+Know the results of your queries on your dialogflow app by PM in Zulip !
+
+{!create-bot-construct-url.md!}
+
+url structure should be like this:
+
+`{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
+
+Modify the parameters of the url above where `api_key` is api key of your Zulip bot \
+and `email` is your zulip email.
+
+* Go to `Fulfillment` setting of your dialogflow app and `enable` Webhooks \
+and provide there the url the webhooks should be sent to.
+
```

Comment:
Okay so:

* Replace `url` with `URL`.
* The same comments on your Beeminder webhook PR apply to this file, so make sure you follow those.
* Capitalize `dialogflow`.
* Change it to `Go to the Fulfillment settings of your Dialogflow app and enable **Webhooks**. Provide the URL constructed above that the notifications should be sent to.`